stats.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Automattic\VIP\Stats;
+
+// Limit tracking to production
+if ( false !== VIP_GO_ENV ) {
+	add_action( 'transition_post_status', __NAMESPACE__ . '\track_publish_post', 9999, 10, 2 );
+}
+
+/**
+ * Count publish events regardless of post type
+ */
+function track_publish_post( $new_status, $old_status ) {
+	if ( 'publish' !== $new_status || 'publish' === $old_status ) {
+		return;
+	}
+
+	$pixel = add_query_arg( array(
+		'v'                     => 'wpcom-no-pv',
+		'x_vip-go-publish-post' => FILES_CLIENT_SITE_ID,
+	), 'http://pixel.wp.com/b.gif' );
+
+	wp_remote_get( $pixel, array( 'blocking' => false, ) );
```

Comment:
Do we need to set a low timeout as well (1s?) just in case the transport that's used doesn't support non-blocking requests?